### ISSUE TITLE
[FEATURE] Show GIES features based on group membership

### ISSUE THREAD

**Issue by dobromirdobrev** [rokwire/illinois-app#1377]
**Re: University Monday 2022/02/21** / 2022/02/22

2 Dobromir/Mladen, Authman Group for GIES. So let's try this please in Dev. Use the Authman Group in Dev seen in the screenshot attached. If I sign in with my netID see if I belong to this group. If a member, show the GIES Onboarding widget, the GIES Twitter widget and the GIES/Canvas widget. You can hide the GIES tile in the Onboarding roles please. You will need to add Dobromir and Todor to this group please Mladen as Admins.

![dev_gies_authman](https://user-images.githubusercontent.com/5978121/155083917-c0dcf988-9632-4b29-8805-686168c9d6c5.png)

**Comment by dobromirdobrev** [comment]
Update 1 (**Re: University Monday 2022/02/21** / 2022/02/23) :

2.1 We will be using 2 Groups for GIES, the data will come from Authman for both of these Groups eventually. For now, in Dev, one will be Authman and the other I made.

2.1.1 The Group in Dev that drives the GIES onboarding widget and the GIES Twitter widget on the home panel is the "Authman test group". I gave you the wrong name yesterday. Can Mladen add me, you and anyone else to this group as Admins from the backend. See screenshot authman_test_group.png.

2.1.2 To drive the Canvas widget on the home panel (Courses) we will use the "Canvas Beta" group. See canvas_beta.png. Mladen can add who we need to this group also.
